build.sh

```diff
@@ -29,7 +29,8 @@ function FirstJsonValue {
 
 # If dotnet CLI is installed globally and it matches requested version, use for execution
 if [ -x "$(command -v dotnet)" ] && dotnet --version &>/dev/null; then
-    export DOTNET_EXE="$(command -v dotnet)"
+    DOTNET_EXE="$(command -v dotnet)"
+    export DOTNET_EXE
 else
```

Comment:
This files is part of template in https://github.com/nuke-build/nuke/blob/6eb779618e22b0cdf1fa27538b305f750b698d88/source/Nuke.GlobalTool/templates/build.sh

Could you please propagate changes there?